subgraphs/aave-forks/protocols/aave-v3/src/constants.ts

```diff
@@ -14,11 +14,12 @@ export const USDC_POS_TOKEN_ADDRESS =
 /////////////////////////////
 
 export namespace Protocol {
+  export const PROTOCOL = "Aave";
   export const NAME = "Aave v3";
   export const SLUG = "aave-v3";
 }
 
-export namespace TokenType {
+export namespace IavsTokenType {
```

Comment:
What is this used for?

Reply:
It is used to determine the reward token type based on whether the token is aToken, vToken, or sToken. see https://github.com/messari/subgraphs/blob/75aa0e594b18c307b660cf798b3e51dead2177de/subgraphs/aave-forks/protocols/aave-v3/src/mapping.ts#L185-L193 . Since schema 3.0.1 already has token.TokenType, I renamed it to IavsTokenType (Input, A, V or S token).